virttest/utils_disk.py

```diff
@@ -20,10 +21,15 @@
 
 from avocado.core import exceptions
 from avocado.utils import process
+from avocado.utils import data_factory
 from avocado.utils.service import SpecificServiceManager
 
 from virttest import error_context
 from virttest import utils_numeric
+from virttest import utils_net
+from virttest import gluster
+from virttest import data_dir
+from virttest.libvirt_xml import pool_xml
```

Comment:
It doesn't make sense to import qemu or libvirt things into this module, otherwise, it will bring circular imports to the future.

Reply:
okay

Reply:
But we are not going to import disk related APIs inside libvirt_xml as they both are totally different, libvirt_xml will only used to edit/generate xml configs where as disk related APIs helps more on functionality.